custom_components/myskoda/coordinator.py

```diff
@@ -228,12 +231,18 @@ async def _on_charging_event(self, event: EventCharging):
         self.set_updated_vehicle(vehicle)
 
     async def _on_access_event(self, event: EventAccess):
+        if self.data.vehicle.status and (timestamp := event.event.timestamp):
+            self.data.vehicle.status.car_captured_timestamp = timestamp
         await self.update_vehicle()
```

Comment:
Wouldn't `update_vehicle` already update this? For an access event a least the timestamp on the event isn't really representative of the `car_capture_timestamp`. That is, the event itself didn't really provide any updated data.